packages/mockyeah-cli/bin/mockyeah-record.js

```diff
@@ -55,7 +55,8 @@ const withName = (env, name) => {
 global.MOCKYEAH_VERBOSE_OUTPUT = Boolean(program.verbose);
 
 boot(env => {
-  const name = program.args[0];
+  const [name] = program.args;
+  const { match } = program;
```

Comment:
Does the inconsistency between array and object whitespace for destructuring bother you?

Reply:
It does, I was cross-eyed last night when I pushed. Have you been formatting with prettier? I needed to turn off auto formatting when I was on the wrong branch. Perhaps now I can turn it back on.

Reply:
It seems prettier doesn't add whitespace to array destructuring.